admin/securedrop_admin/__init__.py

```diff
@@ -375,11 +375,21 @@ def __init__(self, args: argparse.Namespace) -> None:
                 None,
                 lambda config: True,
             ),
+            (
+                "securedrop_app_pow_on_source_interface",
+                True,
+                bool,
+                "On the Source Interface, enable Tor's proof-of-work defense against "
+                "denial-of-service attacks?",
+                SiteConfig.ValidateYesNo(),
+                lambda x: x.lower() == "yes",
+                lambda config: True,
+            ),
             (
                 "securedrop_app_https_on_source_interface",
                 False,
                 bool,
-                "Whether HTTPS should be enabled on " + "Source Interface (requires EV cert)",
+                "On the Source Interface, enable HTTPS? (Requires EV certificate.)",
```

Comment:
Similar question as above: is "Enable HTTPS for the Source Interface (Requires EV certificate.)" clearer?